aiplatform/snippets/create_dataset_test.go

```diff
@@ -65,10 +46,13 @@ func TestCreateDataset(t *testing.T) {
 		t.Errorf("createDataset: wanted 'Create dataset, got %s'", got)
 	}
 
-	output = got
+	output := got
+	teardown(output, t)
 }
 
-func deleteDataset() {
+func teardown(output string, t *testing.T) {
+	t.Helper()
+
 	// parse dataset name--we cannot predict the dataset ID at creation time.
 	tmp := strings.Split(output, "\n")
```

Comment:
optional: consider strings.Cut instead of Split. it has a nicer error condition, IMO.

Reply:
If I'm reading the docs for [`Cut()`](https://pkg.go.dev/strings#Cut) correctly, it looks like it splits a string into two strings on the first instance of the separator. For this resource name, there are 5 instances of the separator and I need the 6th (and final) item.

Reply:
Never mind, I was looking at the wrong place in the code. Done!